cedar-policy/src/ffi/convert.rs

```diff
@@ -225,6 +225,23 @@ pub enum SchemaToJsonAnswer {
     },
 }
 
+/// Convert `cedar_policy::ffi::PolicySet` JSON to `cedar_policy::PolicySet` Object
+pub fn json_to_policyset(
+    json_string: &str,
+) -> Result<crate::PolicySet, Box<dyn std::error::Error>> {
```

Comment:
Returning `Box<dyn Error>` instead of `DetailedError` or something that `impl`s `miette::Diagnostic` diverges from what a lot of the rest of the FFI functions do, and loses structured error information.  Is there a reason this API should have less structured error information (e.g., no help text, no source location) than other APIs?

Reply:
Will change this to match other FFI functions

Reply:
> Could you say a little bit more about why this function is required? As opposed to just using `serde` to deserialize the string into a struct that includes `PolicySet` as a field, and then using `.parse()` on that `PolicySet`

`.parse` for `PolicySet` has visibility `pub (super)` so it is not visible to `CedarJavaFFI`. If changing its visibility to public is acceptable, I can use that instead.

Reply:
I think changing the visibility is preferable to introducing a new function in this case.